modules/cats-effect/src/main/scala/pureconfig/module/catseffect/package.scala

```diff
@@ -229,22 +189,20 @@ package object catseffect {
     *
     * @param conf The configuration to write
     * @param outputStream The stream in which the configuration should be written
-    * @param blocker the blocking context which will be used to load the configuration.
     * @param options the config rendering options
     * @return The return action will save out the supplied configuration upon invocation
     */
   def blockingSaveConfigToStreamF[F[_], A](
```

Comment:
We should remove the deprecated `saveConfigToStreamF` method now, since it has the same signature as this and its implementation doesn't correctly scope blocking operations.